gradle/libs.versions.toml

```diff
@@ -66,3 +67,5 @@ ktxSerializationJson = { module = "org.jetbrains.kotlinx:kotlinx-serialization-j
 # Test
 test-junit = { module = "junit:junit", version.ref = "junit" }
 test-mockk = { module = "io.mockk:mockk", version.ref = "mockk" }
+test-kotlinCompile = { module = "com.github.tschuchortdev:kotlin-compile-testing", version.ref = "compileTesting" }
+test-kotlinCompileKsp = { module = "com.github.tschuchortdev:kotlin-compile-testing-ksp", version.ref = "compileTesting" }
```

Comment:
OMG thank you for this. I feel ashamed that I had not done such a thing before 🤦‍♂️

Honestly, this is best contribution yet, really glad I got to learn this. I should have definitely looked for this before ( i even have a todo to look for this) but I always had something that seemed more important. I should have known though that automated testing is always of the highest priority.

Anyway thank you again!
I was actually fixing this issue locally as well, but I was counting on the weekend to finish it up. I will check the rest of the solution and compare it with mine, maybe I’ll have something to add to the PR.